nuget/Sdk/Sdk.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup Condition="'$(MSBuildProjectExtension)' == '.vcxproj'>
+    <CppWinRTDisableAutoNuGetReference>true</CppWinRTDisableAutoNuGetReference>
```

Comment:
Remove this as it is unrelated tot eh changes in this PR.

Reply:
I would like to respectfully disagree. If this PR is merged without this change applied, then we would have the following situation:

1. A user configures their vcxproj to reference C++/WinRT via NuGet SDK. This does not entail adding the NuGet package to packages.config.
2. The user adds a C++/WinRT file to the project using one of the item templates. The templates [automatically add a packages.config entry](https://github.com/microsoft/cppwinrt/blob/2f2911f8b03eaf4c8f63bab19a69639a7f0e588b/vsix/ItemTemplates/BlankUserControl/cppwinrt_BlankUserControl.vstemplate#L23-L27) when used. Now the user has two references to the C++/WinRT NuGet package (one from the NuGet SDK in this PR, and one using packages.config from the item template). This will cause problems, especially if the two reference have different versions.

PR #633 modifies the item and project templates to automatically detect if the NuGet SDK is in use, and does not add the packages.config reference automatically if that is the case. This is the property the code in that PR uses to detect if the NuGet SDK is used instead of the packages.config file.

If you would prefer, I could move this change to the other PR, but then we'd need to fully merge this PR first before I could reapply the change there, as this file will not exist to modify there until I can rebase it on top of this PR’s changes. I hope this explains things. Thanks!

Reply:
If that's the case we should hold off on merging this until concensus is reached on the other PR.

Reply:
Very well, but I still think this PR and #633 should be reviewed in parallel. They really are two halves of one project.

Reply:
What about merging the PRs?